frameworks/Java/greenlightning/src/main/java/com/ociweb/gl/benchmark/JSONBehaviorInstance.java

```diff
@@ -0,0 +1,38 @@
+package com.ociweb.gl.benchmark;
+
+import com.ociweb.gl.api.GreenRuntime;
+import com.ociweb.gl.api.HTTPRequestReader;
+import com.ociweb.gl.api.HTTPResponseService;
+import com.ociweb.gl.api.RestListener;
+import com.ociweb.json.encode.JSONRenderer;
+import com.ociweb.pronghorn.network.config.HTTPContentTypeDefaults;
+
+public class JSONBehaviorInstance implements RestListener {
+
+
+	private static final JSONRenderer<ResultObject> renderJSON = new JSONRenderer<ResultObject>()
+			.startObject()
+				.string("message", (o,t) -> t.write(o.payload) )
+			.endObject();
```

Comment:
renderJSON is an object which holds no state. its job is to be a 'template' for the given parameterized type.  This object is equiv to  sprintf("{message:%s}"  which is hardcoded and does not change.   The JSONRenderer object has a fluent API allowing any JSON to be defined  where each field is mapped to a lambda for extraction from the given object.

Our design approach in GL was to think about  sprintf("{message:%}" string interpolation and realize that we are making the computer work harder so the human can define the template with %s notation.   Our theory is to make the human define this template in terms of lambdas with a fluent API instead of a block of text.  This may be a little harder for some humans but it is significantly easier for the computer to understand at runtime.

Further any constant string like "{message:%s}" would be cached by the compiler since it was known at startup.  If sprintf were sufficiently smart it could parse these rules once and apply them each time the constant string is passed in.

Reply:
What If I do not create the ResultObject but instead create the RenderJSON object on  each call.  Would this match the rules better?  Seems like RenderJSON has the mapping the rule states that we should create an object mapping the key text to the value text.

Reply:
> This object is equiv to sprintf("{message:%s}" which is hardcoded and does not change.

This is specifically with what we take issue.

> What If I do not create the ResultObject but instead create the RenderJSON object on each call.

I believe that would satisfy our requirements.

Reply:
Ok you can close this pull request. I will submit a new one in the next 48 hours after making this change.

Reply:
There's no need for that - you can just push to your branch and this pull request will reflect that commit.

Reply:
Our updates are ready for review.